### DISCUSSION TITLE
Group 9 discussion community

### DISCUSSION THREAD

**Discussion by Meg-Travers** [Digital-Preservation-Coalition/PythonStudyGroups#5] [General]
Hi Group 9 - post away here!

**Comment by Ish-is-learning** [comment]
Hi Meg! Thanks for setting this up.

**Comment by dozenbart** [comment]
Hello! It's Ellen. Just finished spending some time on Mimo. Looking to try a new resource tomorrow or Friday

**Reply by dozenbart** [reply]
Update: I really like it (as I did the first time, I'm just sticking with it). My main goal is to be sustainable with this group, so I'm going to try to just keep up my streak and will try one or two other resources if my brain can take it.

**Comment by dozenbart** [comment]
Created a sheet for tracking tools if anyone is interested. I also added it to our running notes. Not required obviously, but it's helpful for me to have things in table form. [https://docs.google.com/spreadsheets/d/11UBtfcCzbOKNq55l-FPQJhXDLNCYEcCfBitZQX1N9ZU/edit?usp=sharing](url)

**Comment by k-latta** [comment]
Sorry I missed the meeting tonight. I could not find the Zoom link to save my life.

**Comment by k-latta** [comment]
Here is my script for Thumbs files. I also put my sampling script in here, if anyone wants to see it, or try it out. https://github.com/k-latta/Learning/tree/main

**Comment by Ish-is-learning** [comment]
I can't join for the first hour today because I'll be at a union meeting. I will be online though, so if anyone is having trouble finding the zoom link , let me know.
